frontend/packages/dev-console/src/components/import/serverless/ServerlessSection.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import * as _ from 'lodash';
+import { stateToProps } from '@console/internal/reducers/features';
+import { FLAG_KNATIVE_SERVING } from '@console/knative-plugin/src/const';
```

Comment:
can be imported from main index file directly

Reply:
agreed